<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>937</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(178, 203, 163);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Img_lex">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/lesis.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/FI.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>770</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/logo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/UAT.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>EQUIPO</string>
    </property>
   </widget>
   <widget class="QLabel" name="Img_nat">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/nati.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Img_dodigo">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/dodigo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Img_vic">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>260</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Imagenes/Media/vicky.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Txt_Equipo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>620</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 18pt &quot;Agency FB&quot;;
border :5px solid black;
border-radius: 20px;
</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_lex">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
    color: rgb(255, 255, 255);
background-color: rgb(178, 203, 163);
    border: 2px solid black;
    border-radius: 15px;
}
QCheckBox:hover {
    background-color: rgb(234,82,95) ;
}</string>
    </property>
    <property name="text">
     <string>Lexiss</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_Nat">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
    color: rgb(255, 255, 255);
background-color: rgb(178, 203, 163);
    border: 2px solid black;
    border-radius: 15px;
}
QCheckBox:hover {
    background-color: rgb(234,82,95) ;
}</string>
    </property>
    <property name="text">
     <string>Nati</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_dodigo">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
    color: rgb(255, 255, 255);
background-color: rgb(178, 203, 163);
    border: 2px solid black;
    border-radius: 15px;
}
QCheckBox:hover {
    background-color: rgb(234,82,95) ;
}</string>
    </property>
    <property name="text">
     <string>Dodigo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CB_vic">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
    color: rgb(255, 255, 255);
background-color: rgb(178, 203, 163);
    border: 2px solid black;
    border-radius: 15px;
}
QCheckBox:hover {
    background-color: rgb(234,82,95) ;
}</string>
    </property>
    <property name="text">
     <string>Vicky</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
